File: Fibonacci/Fibonacci.java
public class Fibonacci {
    public static int fibo(int n) {
        if (n < 2) {
            return n;
        } else {
            return fibo(n - 1) + fibo(n - 2);
        }
    }

    public static void main(String args[]) {
        double startTime = (double) System.nanoTime();

        for (int i=0; i < 1000; ++i) {
            fibo(28);
        }

        double stopTime = (double) System.nanoTime();
        System.out.println("Java " + ((stopTime - startTime) / 1000000 / 1000) + " ms");
    }
}

File: Fibonacci/fibonacci.js
function fibo(n) {
    if (n < 2)
        return n;
    else
        return fibo(n - 1) + fibo(n - 2);
}

var t0 = process.hrtime();

for (var i = 0; i < 125; ++i) {
    fibo(28);
}

var stop = process.hrtime(t0);

mean = stop[1] / 1000000;  // process.hrtime(start_time) returns [seconds, nanoseconds]
mean /= 125;

console.log(`NodeJS ${mean} ms`);